### PR TITLE
PXP-7415 Feat/product refactor

### PR DESCRIPTION
**Pull request by mfshao**
This PR fulfills https://ctds-planx.atlassian.net/browse/PXP-7415

Refactor gen3.org's product page to break it down into 3 components: "Framework Services", "Data Commons", "Workspaces"

### New Features
- Refactor gen3.org's product page to break it down into 3 components: "Framework Services", "Data Commons", "Workspaces"

### CODE REVIEW

**Review by wangfan860 — APPROVED** [review]
👍👍

**Review by xritter1 — APPROVED** [review]

**Comment by mfshao** [comment]
Awaiting text updates

File: content/products/workspaces.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+identifier: products/workspaces
+title: Gen3 - Products - Workspaces
+linktitle: /products/workspaces
+date: 2021-01-08
+g3Teaser:
+  title: Workspaces
```
Comment (mfshao) [inline content/products/workspaces.md:7]: call this `Gen3 Workspaces` as well?

File: content/products/framework-services.md [hunk 2]
```diff
@@ -4,21 +4,22 @@ title: Gen3 - Products - Framework Services
...
+    detail: The Gen3 data object indexing service mints permanent digital IDs, called data GUIDs (globally unique IDs), for data objects. The service tracks the physical locations and hash of every asset (file) in the data commons object store. The Gen3 platform includes landing pages which support FAIR descriptions of the data objects.
   f2:
-    title: AuthN/AuthZ
+    title: User Authentication and Authorization
     fig: /figs/features/auth.svg
-    detail: Gen3 utilizes OpenID Connect for providing AuthN services with AuthZ specified on a per commons basis. Currently supported identity providers include Google and Shibboleth, supporting providers such as NIH iTrust, InCommon Federation, and eduGAIN.
+    detail: The Gen3 Framework Services include software services for user authentication and fine-grained authorization to services, metadata, and data objects via an attribute-based access control (ABAC) policy engine. Gen3 utilizes OpenID Connect for providing AuthN services with AuthZ specified on a per commons basis. Currently supported identity providers include Google and Shibboleth, supporting providers such as NIH iTrust, InCommon Federation, and eduGAIN.
```
Comment (xritter1) [inline content/products/framework-services.md:18]: I don't know if this is not obvious to all, but: 
"Gen3 utilizes OpenID Connect for providing Authentication (AuthN) services with Authorization (AuthZ) specified on a per commons basis. Currently supported identity providers include Google and Shibboleth, supporting providers such as NIH iTrust, InCommon Federation, and eduGAIN."

File: content/products/workspaces.md [hunk 3]
```diff
@@ -5,20 +5,22 @@ linktitle: /products/workspaces
 date: 2021-01-08
 g3Teaser:
   title: Workspaces
-  detail: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+  detail: Gen3 workspaces are secure data analysis environments in the cloud that can access data from one or more data resources, including Gen3 Data Commons. Workspaces are often fully integrated with a specific data commons, and coming soon are workspaces as stand-alone analysis environments with a user pay model.
+  para1: Gen3 Workspaces use the Gen3 Framework Services for user authentication and authorization and for retrieving data objects and metadata from data resources, like Gen3 Data Commons.
+  para2: By default Gen3 Workspaces include Jupyter notebooks and RStudio. Workspaces can be further configured to offer a variety of analysis environment "flavors", pre-loaded with various custom applications. Virtually any application can be integrated into a Gen3 Workspace, like analysis workflows, data processing pipelines, or data visualization apps.
```
Comment (xritter1) [inline content/products/workspaces.md]: To shorten it a bit if wanted: The third section is repeated below, too:
"By default Gen3 Workspaces include Jupyter notebooks and RStudio, but virtually any application such as analysis workflows, data processing pipelines, or data visualization apps can be integrated into a Gen3 Workspace."

File: content/products/workspaces.md [hunk 4]
```diff
@@ -5,20 +5,22 @@ linktitle: /products/workspaces
...
   f1:
     title: RStudio
     fig: /figs/features/rstudio.svg
-    detail: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+    detail: Gen3 Workspaces come pre-configured with RStudio, a development environment for the statistical programming language R. Scripts saved in the workspace persist after logging out and can be shared with colleagues for collaborative analyses.
   f2:
     title: Jupyter Notebooks
     fig: /figs/features/jupyter.svg
-    detail: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+    detail: Gen3 Workspaces come with the ability to create, upload, modify and run Jupyter Notebooks by default, which are interactive documents that allow you to create and share live code, equations, visualizations and narrative text.
```
Comment (xritter1) [inline content/products/workspaces.md:19]: "Gen3 Workspaces come with the ability to create, upload, modify, and run Jupyter Notebooks by default, which are interactive documents that allow you to create and share live code, equations, visualizations, and narrative text."

File: content/products/workspaces.md (hunk 3 above)
Comment (cgmeyer) [reply content/products/workspaces.md]: done, thanks for the R

**Pull request merged**